src/hooks/useEnsName.ts

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from 'react'
+
+export const useEnsName = (provider, address) => {
+  const [ensName, setEnsName] = useState('')
```

Comment:
See https://github.com/pooltogether/pooltogether-react-components/pull/5#pullrequestreview-717483357.

We could just use `useReadProvider(NETWORK.mainnet)` if we only want to lookup ENS on mainnet